src/components/fields/ArrayField.js

```diff
@@ -101,7 +101,7 @@ class ArrayField extends Component {
       event.preventDefault();
       this.asyncSetState({
         items: this.state.items.filter((_, i) => i !== index)
-      }, {validate: true}); // refs #195
```

Comment:
Hmm sorry about that, I wasn't seeing that bug when I was testing yesterday, but now it's there. I must have missed a step.